src/test/e2e/cypress/e2e/footer.cy.js

```diff
@@ -18,4 +18,10 @@ it('should redirect to impressum', () => {
     cy.get("footer").find("a").eq(1)
         .should('have.attr', 'href')
         .and('include', 'https://www.unilu.ch/impressum/');
-});
\ No newline at end of file
+});
+
+it('should redirect to github-page', () => {
```

Comment:
The test name is unfitting as it only checks for the href attribute and not if the redirect actually happens.